### PR TITLE
Updates to utilization-bar-chart

### PR DESCRIPTION
**Pull request by michael-coker**
Updates to copy per our feedback discussion.

### CODE REVIEW

File: pattern-library/data-visualization/utilization-bar-chart/design/design.md [hunk 1]
```diff
@@ -2,27 +2,20 @@
...
-    - The used value bar shows the percentage (out of 100%) that has been consumed. The width and color of this bar can change according to the data.
-    - **Chart Fill:**
-      - The fill color can change to indicate that the data has surpassed a threshold. These include an orange warning and red error threshold, which can be used to indicate to a user that their consumption is nearing capacity.
-      - When no thresholds are specified, the used value bar is blue.
-      - When thresholds exist but have not been surpassed, the used value bar is green.
-
-  1. **Total Value**
-    -The total value bar is static in width and color. It represents the total available amount of a resource, and serves as a point of reference for the user as they study the chart.
+1. **Used Value:** The used value bar shows the percentage (out of 100%) that has been consumed. The width and color of this bar can change according to the data.
+    - **Chart Fill:** The fill color can change to indicate that the data has surpassed a threshold. These include an orange warning and red error threshold, which can be used to indicate to a user that their consumption is nearing capacity.
```
Comment (LHinson) [inline pattern-library/data-visualization/utilization-bar-chart/design/design.md]: Do you think this should be called "Threshold (optional)" instead?

File: pattern-library/data-visualization/utilization-bar-chart/design/design.md [hunk 2]
```diff
@@ -2,27 +2,20 @@
...
+        - When no thresholds are specified, the used value bar is blue (#39a5dc).
+        - When thresholds exist but have not been surpassed, the used value bar is green (#3f9c35).
+  1. **Total Value:** The total value bar is static in width and color. It represents the total available amount of a resource, and serves as a point of reference for the user as they study the chart.
       - The background color is grey (#d1d1d1).
-
-  1. **Tooltip:** The chart does not effectively show precise values in its original state, so it is recommended that you display a tooltip on hover with the exact percentage value of the used value.
+  1. **Tooltip:** The chart does not effectively show precise values in its original state, so it is recommended that you display a tooltip on hover with the exact percentage used.
 
 ## **Warning Threshold (optional)**
```
Comment (LHinson) [inline pattern-library/data-visualization/utilization-bar-chart/design/design.md]: If we put "Threshold (optional)" above, maybe we can remove "(optional)" in the title here? Maybe this whole section is called Threshold Examples with Warning and Error underneath? WDYT?

File: pattern-library/data-visualization/utilization-bar-chart/design/design.md (hunk 1 above)
Comment (michael-coker) [reply pattern-library/data-visualization/utilization-bar-chart/design/design.md]: @LHinson Hmm... that bullet is addressing fill colors, and mentions the fill colors of thresholds. How about we update it to something like this?

    - **Chart Fill:** The default used value fill color is blue (#39a5dc).
        - **Thresholds** (optional): The fill color can change to indicate that the used value has surpassed a threshold. These include orange (#ec7a08) warning and red (#cc0000) error thresholds, which can be used to indicate to a user that their consumption is nearing capacity. When thresholds exist but have not been surpassed, the used value fill color is green (#3f9c35).

screen shot 2018-02-21 at 4 13 41 pm

File: pattern-library/data-visualization/utilization-bar-chart/design/design.md (hunk 2 above)
Comment (michael-coker) [reply pattern-library/data-visualization/utilization-bar-chart/design/design.md]: @LHinson I like it. Reflected that change in the screenshot in my previous comment.

**Comment by LHinson** [comment]
Oy, sorry for the delay @michael-coker! Yes, I think the screenshot looks great and like your adjustments!

**Comment by LHinson** [comment]
@michael-coker this is still waiting for updates to reflect that screenshot, correct? Just want to confirm I'm not blocking this! LMK and thanks

**Comment by michael-coker** [comment]
@LHinson oops! Thanks for the nudge, this fell off my radar. Committed and updated the branch.

**Comment by LHinson** [comment]
bitmoji

**Pull request merged**